### ISSUE TITLE
list index out of line when using 'nosetup'?

### ISSUE THREAD

**Issue by bmarcote** [jive-vlbi/sched#24]
I think that pySCHED runs over an index-out-of-range error when using no setup.
The old NRAO SCHED allows the possibility to run a schedule with no specific setup (useful e.g. when a setup is not yet available for the desired antennas) via typing 'no setup' (and no 'setup =' line in the rest of the key file).

When doing this in a working schedule file, it shows the following error when running:

```
SRREAD:  Reading source catalog:  /Users/hawky/.pysched/catalogs/sources.gsfc
INPUT:   Found    21 input scans.
SHORTN: WARNING - Source names longer than 8 characters used.
        This will be a problem if geodetic software (SOLV etc) is used.
        This might be a problem on some correlators.
Error running pySCHED: list index out of range
Traceback (most recent call last):
  File "/Users/hawky/Library/Python/3.10/bin/sched.py", line 152, in <module>
    defaults()
  File "/Users/hawky/Library/Python/3.10/lib/python/site-packages/pysched/sched/defaults_module.py", line 58, in defaults
    bps = setups[station.nsetup[scan_index] - 1].totbps
IndexError: list index out of range
```

Hence I think that pySCHED is still searching for a given setup in the database.

I can provide you in private the key file I used, although it should be replicable with any key file just by changing the 'setup' line.

**Comment by bmarcote** [comment]
I actually encountered this these days again. I was going to put an issue but I apparently did last year for the same session :-) 

Just to correct that the keyword to write is: `nosetup`, instead of `no setup`.

**Comment by bobeldering** [comment]
This bug was triggered when GRABTO is used, DATAPATH has the value 'IN2DISK' and GRABGAP is undefined (or has the value 0).
In this case (py)SCHED would try to default the value of GRABGAP, as documented, to:  5.0 + GRABTIME(1)*bitrate/110Mbps 

Obviously, to get the bitrate requires a setup, which would fail in case of NOSETUP. This code was translated from Fortran SCHED. However, the value of GRABGAP is never used. So, the solution was to just remove the code that computes the default of GRABGAP. This is implemented in 1.26.4.
